SwiftWisdom/Core/UIKit/UILabel+Typography.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  UILabel+Typography.swift
+//  SwiftWisdom
+//
+//  Created by Patrick Butkiewicz on 4/15/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UILabel {
+    
+    /**
+     Sets the text of the UILabel instance to an attributed string with tracking applied
+     - Parameter value: The value in points that each character is kerned to
+     */
+    
+    public func ip_setCharacterSpacing(value: CGFloat) {
+        addAttribute(NSKernAttributeName, value: value)
+    }
+    
+    /**
+     Sets the text of the UILabel instance to an attributed string with the specified line spacing
+     - Parameter value: The space, in points, between each line.
+     */
+    
+    public func ip_setLineSpacing(value: CGFloat) {
+        let paragraphSpacing = NSMutableParagraphStyle()
+        paragraphSpacing.lineSpacing = value
+        addAttribute(NSParagraphStyleAttributeName, value: paragraphSpacing)
```

Comment:
I'm not sure it would override existing paragraph styles, unless they were also defined on the whole string. Theoretically, this should just add another attribute around the entirety of the screen. I'm not sure how it would affect conflicting paragraph style properties though. The other option would be to enumerate all attributes, identify paragraph styles and set their line spacing attribute.